backend/inspirehep/factory.py

```diff
@@ -22,6 +23,29 @@ def config_loader(app, **kwargs_config):
     return result
 
 
+class DispatcherMiddlewareWithoutHostNameChecking(DispatcherMiddleware):
+    """WSGI middleware without checking hostname.
+
+    It will override the request ``HOST`` to always match
+    the ``SERVER_NAME``.
+    """
+
+    def __call__(self, environ, start_response):
+        if "SERVER_NAME" in environ:
+            environ["HTTP_HOST"] = environ["SERVER_NAME"]
+        return super().__call__(environ, start_response)
+
+
+def inspirehep_create_wsgi_factory(mounts_factories):
```

Comment:
is the `inspirehep_` prefix really needed?

Reply:
just not to make it clear is not the default invenio one